stubs/iterable.stub

```diff
@@ -275,3 +275,24 @@ class RecursiveArrayIterator extends ArrayIterator implements RecursiveIterator
      */
     public function uksort($cmp_function) { }
 }
+
+/**
+ * @template-covariant TKey
+ * @template-covariant TValue
+ * @template TIterator as Iterator<TKey, TValue>
+ *
+ * @template-extends IteratorIterator<TKey, TValue, TIterator>
+ *
+ * @mixin TIterator
+ */
+class LimitIterator extends \IteratorIterator
```

Comment:
Needs to be added here: https://github.com/phpstan/phpstan-src/blob/2b3d28244afa01be2dc7df71e6879bc5fd99b1b5/conf/config.neon#L25-L29

Then you probably won't get that extra error.

Reply:
Done, but the error remains. I assume this is because  the generic is incomplete instead of missing?